doc/tutorials/content/sources/function_filter/sphere_removal.cpp

```diff
@@ -0,0 +1,51 @@
+#include <pcl/common/generate.h>
+#include <pcl/filters/experimental/functor_filter.h>
+#include <pcl/point_types.h>
+
+#include <iostream>
+
+int
+main(int argc, char** argv)
+{
+  using XYZCloud = pcl::PointCloud<pcl::PointXYZ>;
+  const auto cloud = pcl::make_shared<XYZCloud>();
+  const auto filtered_cloud = pcl::make_shared<XYZCloud>();
+
+  // Create a random generator to fill in the cloud
+  pcl::common::CloudGenerator<pcl::PointXYZ, pcl::common::UniformGenerator<float>>
+      generator{{-2.0, 2, 1234}};
+  generator.fill(10, 1, *cloud);
+
+  std::cerr << "Cloud before filtering: " << std::endl;
+  for (const auto& pt : *cloud)
+    std::cerr << "    " << pt.x << " " << pt.y << " " << pt.z << std::endl;
+
+  // Setup a condition to reject points inside a filter
+  Eigen::Vector3f center{0, 0, 2};
+  float radius = 2;
+
+  pcl::experimental::SimpleFilterFunction<pcl::PointXYZ> filter;
+  filter = [=](const XYZCloud& cloud, pcl::index_t idx) {
+    if ((cloud[idx].getVector3fMap() - center).norm() < radius) {
```

Comment:
`return ((cloud[idx].getVector3fMap() - center).norm() >= radius);` would work, too.

Reply:
Maybe use `squaredNorm()` and `SquaredRadius` ? Also, can't the center and radius be declared const?

Reply:
Will update to use norm, >= and const the radius. This is just an example, so wanted to make it a bit approachable